### DISCUSSION TITLE
Camera Intrinsics

### DISCUSSION THREAD

**Discussion by andy-96** [bdd100k/bdd100k#139] [Q&A] [answered]
Hello together,

is there any camera intrinsics available for the dataset? [Here](https://doc.bdd100k.com/format.html), intrinsics are mentioned however, I could not find them in the detection labels. Could you help me out?

Cheers

**Comment by fyu** [comment, marked as answer]
The videos are captured by iPhone 5 by 720p videos. You can calculate approximate intrinsic parameters from the hardware spec. We usually use focal length 700px as initialization for structure from motion.

**Comment by vishnubanna** [comment]
@andy-96  did you end up finding the intrinsics matrix?

**Reply by andy-96** [reply]
Hi Vishnu,
actually no, in my model, I am predicting the intrinsics with an extra head similar to this one: https://github.com/google-research/google-research/blob/0487481e70e99f44754ffb51584a212c19ce0a05/depth_from_video_in_the_wild/motion_prediction_net.py#L23

But with @fyu answer above, it should not be that hard to figure out approximate intrinsic parameters.
